src/main/charts/confluence/values.yaml

```diff
@@ -183,6 +183,18 @@ synchrony:
     periodSeconds: 1
     # -- The number of consecutive failures of the Synchrony container readiness probe before the pod fails readiness checks
     failureThreshold: 30
+  resources:
+    jvm:
+      # -- The maximum amount of heap memory that will be used by the Synchrony JVM
+      maxHeap: "1g"
+      # -- The minimum amount of heap memory that will be used by the Synchrony JVM
+      minHeap: "1g"
+      # -- The memory allocated for the Synchrony stack
+      stackSize: "2048k"
+    container: 
+      requests:
+        cpu: "2"
+        memory: "2G"
```

Comment:
@badgersow do we need 2CPU and 2G of memory for a single synchrony unit? It seems pretty hungry.